test/simulator/main/org/apache/cassandra/simulator/paxos/PairOfSequencesAccordSimulation.java

```diff
@@ -150,6 +152,13 @@ public PairOfSequencesAccordSimulation(SimulatedSystems simulated,
         this.validator = validator;
     }
 
+    @Override
+    protected Class<? extends Throwable>[] expectedExceptions()
+    {
+        return (Class<? extends Throwable>[]) new Class<?>[] { RequestExecutionException.class,
+                                                               CoordinationFailed.class };
```

Comment:
paxos test doesn't fail if `RequestExecutionException` are seen, which accord inherited.  But Accord has its own error types, so need to allow `CoordinationFailed`.  We could try to shrink this list, but figured if this was fine for paxos this is fine here?